.env.example

```diff
@@ -33,6 +33,7 @@ KEYCLOAK_HOST_HTTPS_PORT=8443
 #KEYCLOAK_IMAGE=
 HAPROXY_HOST_HTTP_PORT=80
 HAPROXY_HOST_HTTPS_PORT=443
+HAPROXY_RGW_HOST_HTTP_PORT=90
```

Comment:
Lets use a range of custom ports rather than 90

Reply:
A custom port number is any port number outside of the standard or well-known ports (0-1023).

Reply:
updated it to 9080